.gitignore

```diff
@@ -6,7 +6,7 @@ docs/api.json
 docs/changelog.md
 docs/readme.md
 git-plugins
-
+plugins/test_plugin.py
```

Comment:
we could add e local_plugins folder to gitignore and default config of the plugin runner to allow easy development with local plugins.